Kernel/Arch/x86/CPU.h

```diff
@@ -33,7 +33,7 @@ inline u32 get_iopl_from_eflags(u32 eflags)
 const DescriptorTablePointer& get_gdtr();
 const DescriptorTablePointer& get_idtr();
 
-[[noreturn]] void handle_crash(RegisterState&, const char* description, int signal, bool out_of_memory = false);
+[[noreturn]] void handle_crash(RegisterState const&, const char* description, int signal, bool out_of_memory = false);
```

Comment:
Can you also change `description` to east const while you're editing that line? :)

Reply:
sure